Source/WebInspectorUI/UserInterface/Views/LogContentView.css

```diff
@@ -61,6 +56,56 @@
     color: var(--glyph-color-active-pressed);
 }
 
+.navigation-bar > .item.find-banner.console {
+    flex-wrap: nowrap;
+    position: relative;
+    top: auto;
+    border: none;
+}
+
+:not(.console-drawer) > .navigation-bar > .item.find-banner.console {
+    margin-inline-start: var(--find-banner-inline-space);
+}
+
+.navigation-bar > .item.find-banner.console > input[type="search"] {
+    padding-top: 0;
+    border: 1px solid var(--border-color);
+    outline: none;
+
+    --border-radius-start: 3px;
+    --border-radius-end: 0;
+}
+
+body[dir=ltr] .navigation-bar > .item.find-banner.console > input[type="search"] {
+    border-top-left-radius: var(--border-radius-start);
+    border-bottom-left-radius: var(--border-radius-start);
+    border-top-right-radius: var(--border-radius-end);
+    border-bottom-right-radius: var(--border-radius-end);
+}
+
+body[dir=rtl] .navigation-bar > .item.find-banner.console > input[type="search"] {
+    border-top-left-radius: var(--border-radius-end);
+    border-bottom-left-radius: var(--border-radius-end);
+    border-top-right-radius: var(--border-radius-start);
+    border-bottom-right-radius: var(--border-radius-start);
+}
```

Comment:
I think we can use the logical border radius properties here. e.g. `border-start-start-radius` (https://developer.mozilla.org/en-US/docs/Web/CSS/border-start-start-radius)